### PR TITLE
Common commands translated to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by schneiderl**
Added the translation of a few Common commands.

### CODE REVIEW

File: pages.pt-BR/common/wpa_supplicant.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# wpa_supplicant
+
+> Gerenciador de redes wireless protegidas.
+
+- Entrar em uma rede wireless protegida:
+
+`wpa_supplicant -i {{interface}} -c {{caminho/para/wpa_supplicant_conf.conf}}`
+
+- Entrar emuma rede wireless protegita e executar em um daemon:
```
Comment (waldyrious) [inline pages.pt-BR/common/wpa_supplicant.md]: Uh-oh, a typo sneaked in here ;) I'll submit a follow-up PR to fix.

Comment (schneiderl) [reply pages.pt-BR/common/wpa_supplicant.md]: Yeah,
Nice that you noticed it.

Comment (waldyrious) [reply pages.pt-BR/common/wpa_supplicant.md]: As they say... "Many eyes make bugs shallow" :heart:

Comment (waldyrious) [reply pages.pt-BR/common/wpa_supplicant.md]: By the way, the update PR was #2889.

**Pull request merged**